cfme/tests/cloud/test_stack.py

```diff
@@ -61,6 +62,10 @@ def test_resources_link(stack):
 
 def test_edit_tags(stack):
     stack.edit_tags("Cost Center *", "Cost Center 001")
+    row = sel.elements("//*[(self::th or self::td) and normalize-space(.)={}]/../.."
```

Comment:
I think that perhaps this functionality should move back into the previous file, but maybe under a method named, ```get_tags```. We should be avoiding having ```sel``` calls in tests, unless we can't really avoid it. But in my opinion, getting the tags is a legitimate method for a Stack.